components/exercises/chapter-5-oscar/DominantDirection/getDirection.tsx

```diff
@@ -0,0 +1,40 @@
+import characterScript from '@/constants/chapter-5/characterScript';
+import countBy from '@/constants/chapter-5/countBy';
+
+type CharacterScript = {
+  name: string;
+  count: number;
+};
+
+/**
+ * Determines the dominant direction of a given text.
+ * @param text - The text to analyze.
+ * @returns The dominant direction of the text.
+ */
+export function getDirection(text: string): string {
+  // count the number of characters that belong to each script
+  let counted: CharacterScript[] = countBy(Array.from(text), (char) => {
```

Comment:
It's a good practice to declare variables that aren't reassigned later in the program with `const`. By declaring them with `let` we are implying that `counted` will be reassigned later. Same for other variables below.

Note: Usually this would've been caught by this eslint rule https://eslint.org/docs/latest/rules/prefer-const but I think we don't have it enabled yet